pkg/iris/scan.go

```diff
@@ -180,7 +181,11 @@ func (s *scanner) CreateDiscoveryEvent(namespace corev1.Namespace, deployments [
 
 	// Metadata for the event
 	class := fmt.Sprintf("discoveryItem/service/kubernetes")
-	id := fmt.Sprintf("%s/%s", class, scope)
+
+	idString := fmt.Sprintf("%s/%s", class, scope)
+	sum := sha256.Sum256([]byte(idString))
+
+	id := hex.EncodeToString(sum[:])
```

Comment:
Do we need the ":" here?

Reply:
@DennisBerthold-leanix yeah, it can't convert directly from byte array so we have to slice it

Reply:
also to ensure UTF-8 encoding